django/__init__.py

```diff
@@ -22,3 +27,38 @@ def setup(set_prefix=True):
             '/' if settings.FORCE_SCRIPT_NAME is None else settings.FORCE_SCRIPT_NAME
         )
     apps.populate(settings.INSTALLED_APPS)
+
+
+def setup(set_prefix=True):
+    """
+    This function is idempotent (calling it several times changes nothing), but
```

Comment:
The docstring here needs a little reworking... 

At least the `Configure the settings (this happens as a side effect of accessing the first setting),` part of the existing docstring should come here.